src/main/views/sso-user-successful.njk

```diff
@@ -0,0 +1,39 @@
+{% from "macros/breadcrumb.njk" import breadcrumb %}
+{% from "macros/csrf.njk" import csrfProtection %}
+{% from "macros/errorSummary.njk" import errorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% extends "template.njk" %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ breadcrumb([
+    ['Manage users', urls.MANAGER_USERS_URL],
+    ['Remove SSO']
+  ]) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+      <div class="govuk-grid-column-three-quarters">
+        {{ govukPanel({ titleText: "Single sign on has been removed" }) }}
```

Comment:
Think it should be 'Single sign-on' not 'Single sign on'

Reply:
Also the title should be 'Single sign-on removed successfully' to keep it consistent with 'delete user successful' and 'remove user successful' views

Reply:
fixed